.github/workflows/black.yml

```diff
@@ -15,5 +15,5 @@ jobs:
       - uses: actions/checkout@v4
       - uses: psf/black@stable
         with:
-          options: "--check --verbose"
+          options: "--check --verbose --line-length 88"
```

Comment:
this is great.  Can you put a note in the `docs/for_developer.rst` file that "We use Black's default line length of 88 characters."?

Reply:
Done!